client/sync_int_test.go

```diff
@@ -0,0 +1,93 @@
+// eventsgateway
+// +build integration
+// https://github.com/topfreegames/eventsgateway
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2019 Top Free Games <backend@tfgco.com>
+
+package client_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/topfreegames/eventsgateway/app"
+	"github.com/topfreegames/eventsgateway/client"
+	extensions "github.com/topfreegames/extensions/kafka"
+	"google.golang.org/grpc"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Sync Client", func() {
+	var (
+		a        *app.App
+		c        *client.Client
+		consumer *extensions.Consumer
+	)
+	name := "EventName"
+	props := map[string]string{
+		"prop1": "val1",
+		"prop2": "val2",
+	}
+
+	BeforeEach(func() {
+		var err error
+		consumer, err = extensions.NewConsumer(config, logger)
+		Expect(err).NotTo(HaveOccurred())
+		go consumer.ConsumeLoop()
+		time.Sleep(1 * time.Second) // wait consumer receive assign partition
```

Comment:
(out of scope/suggestion) It would be nice if `consumer` had a `WaitUntilReady(timeout)` method, so we don't have to do this sleep all over. Plus, it's likely that it takes less than a second to be ready, but we're having each test wait for at least one second, which adds to the time it takes to run all tests.

Reply:
done